CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportDelimitedPlugin.java

```diff
@@ -112,6 +113,12 @@ public PluginParameters createParameters() {
         parserParameters.setDescription("The PluginParameters used by the parser");
         params.addParameter(parserParameters);
 
+        final PluginParameter<BooleanParameterValue> filesIncludeHeadersParam = BooleanParameterType.build(FILES_INCLUDE_HEADERS_PARAMETER_ID);
+        filesIncludeHeadersParam.setName("files Include Headers");
+        filesIncludeHeadersParam.setDescription("True if the files include headers in the first row, default is True");
```

Comment:
I'd probably omit everything after the comma as the default value doesn't really need an explicit mention here

Reply:
This option is set to True by default, does this have a different meaning?

Reply:
I guess this is not used in the GUI anyway, but it'll be, if we expose them through the API someday.

Reply:
Something like that yes

Reply:
Fixed for the schemaParam as well, to be consistent...